app/views/groups/index.html.erb

```diff
@@ -0,0 +1,30 @@
+<%= render 'shared/navbar', group: @groups %>
+<button class="back"><%= link_to "back", groups_path  %></button>
+
+<div class="group_main_div">
+<h2 class="group_main_title">All Category</h2>
+
+<section class="main_group_section">
+<div class="groups_section">
+  <% @groups.each do |group| %>
+    <div>
```

Comment:
- [ ] Dear @lawrahkonwea, though your functionality is working fine, Please keep in mind this is a mobile application, and as such it is expected to be working well with the mobile design, I can see your styling is not behaving well with a mobile design. As you can see, when I add an image to the category it overflows and does not fit well with the design. Please consider the mobile layout and rearrange your styles. 

### Your mobile design 
![image](https://github.com/lawrahkonwea/Budget_APP/assets/69077061/dd93b098-3b22-41b1-9f3a-b96f1a73ca6e)